### PR TITLE
Clarifying note for need to approve CSRs

### PR DESCRIPTION
**Pull request by alokstech**
Customers are getting confused with the note and assuming that the cluster will start fine with no manuals steps untill he expiry date of  kube-apiserver-to-kubelet-signer. However, the CSRs need to be aproved when starting the cluster if the kubelet certificates are expired in the nodes.

Version(s):

4.17, 4.16, 4.15, 4.14

Issue:

https://issues.redhat.com/browse/OCPBUGS-47550

Link to docs preview: https://86600--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html#graceful-shutdown_graceful-shutdown-cluster

https://86600--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/graceful-shutdown.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ You can shut down your cluster in a graceful manner so that it can be restarted
 
 [NOTE]
 ====
-You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire.
+You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire. However, you might have to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates when the cluster restarts.
```
Comment (xenolinux) [inline modules/graceful-shutdown.adoc:13]:
```suggestion
You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire. However, you might need to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates when the cluster restarts.
```

**Review by xenolinux — COMMENTED** [review]
A tiny nit; otherwise LGTM

I added /ok-to-test. Once preview is generated, you can add the preview link in the comment 0 and get a QE approval before the merge review. Thanks.

**Comment by alokstech** [comment]
@xenolinux Could you please help to add the QE contact here?

**Comment by xenolinux** [comment]
> @xenolinux Could you please help to add the QE contact here?

@alokstech I asked Ge to review your PR. FYI: Next time, please ping on the `forum-openshift-qe` slack channel for QE approvals. Thanks!

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Pull request merged**